build_library/toolchain_util.sh

```diff
@@ -172,12 +174,19 @@ get_sdk_libdir() {
 get_sdk_binhost() {
     local arch=$(get_sdk_arch) ver
     if [[ $# -eq 0 ]]; then
-        set -- "${FLATCAR_SDK_VERSION}" "${FLATCAR_VERSION_ID}"
+        set -- "${FLATCAR_SDK_VERSION}"
     fi
 
     for ver in "$@"; do
-        echo "${FLATCAR_DEV_BUILDS}/sdk/${arch}/${ver}/pkgs/"
+        # Use this order because usually only crossdev needs to be fetched
+        # from /toolchain/ in the setup_board step.
+        # Also prefer /toolchain-arm64/ crossdev Rust over /pkgs/ Rust if
+        # an arm64-usr board is setup.
+        # The last is /pkgs/ if something needs to be reinstalled in the SDK
+        # but normally it's not needed.
         echo "${FLATCAR_DEV_BUILDS}/sdk/${arch}/${ver}/toolchain/"
+        echo "${FLATCAR_DEV_BUILDS}/sdk/${arch}/${ver}/toolchain-arm64/"
```

Comment:
I thought it would be a good idea to have the crossdev Rust available to be installed directly though emerge and not only hidden behind the update chroot action. I'll delete it again so that it is only used when the crossdev Rust is installed.

Reply:
Done